do

```diff
@@ -15,211 +15,450 @@ from epimodel.gleam import Batch, batch
 log = logging.getLogger("do")
 
 
-def import_countermeasures(args):
-    log.info(f"Importing countermeasures from {args.SRC} into {args.DEST} ...")
-    cms = epimodel.imports.import_countermeasures_csv(args.rds, args.SRC)
-    cms.to_csv(args.DEST)
-    log.info(
-        f"Saved countermeasures to {args.DEST}, {len(cms.columns)} features, "
-        f"last day is {cms.index.get_level_values(1).max()}"
+# Global arguments
+
+
+@click.group()
+@click.option("-d", "--debug", is_flag=True, help="Enable debugging logs.")
+@click.option(
+    "-C",
+    "--config",
+    type=click.Path(exists=True),
+    default="config.yaml",
+    envvar="EPI_CONFIG",
+    help="Path to config file (default config.yaml; alternatively set EPI_CONFIG environment variable).",
+)
+@click.pass_context
+def cli(ctx, debug, config):
+    """
+    Epimodel pipeline runner.
+
+    See https://github.com/epidemics/epimodel for more details.
+
+    1. Update Johns Hopkins data:
+    
+    ./do action update-johns-hopkins (not needed if you got fresh data from the repo)
+
+    2. Generate batch file from estimates and basic Gleam XML definition. 
+    
+    ./do action generate-gleam-batch -D 2020-04-15 -c JK default.xml estimates-2020-04-15.csv
+
+    The batch file now contains all the scenario definitions and initial populations. Note the estimate input specification may change.
```

Comment:
These look rather longer - can we split it into multiple lines to fit in the black's 88 chars line length? Similar with other docstrings

Reply:
Wrapped (I think... is there an option for Black to check for this?)